setup.py

```diff
@@ -30,7 +29,7 @@
     package_dir={"": "src"},
     include_package_data = True,
     packages=setuptools.find_packages(where="src"),
-    python_requires=">=3.6",
-    install_requires=['numpy', 'pandas', 'xarray', 'toml', 'scipy'],
+    python_requires=">=3.8",
+    install_requires=['numpy', 'pandas>=1.5.0', 'xarray', 'toml', 'scipy'],
```

Comment:
Should we go ahead and define `>=` for the other packages as well?

Reply:
At least for `xarray`, `scipy` and `numpy` I would say.

Reply:
On my local setup, which is made using `conda` and following the recipe, I have `xarray==2022.11.0`, `scipy==1.9.3` and `numpy: 1.23.4`

Reply:
I have `xarray==2022.6.0`, `scipy==1.9.1` and  `numpy==1.23.2`, which look like earlier versions than what you are using @patrickjwright.

So we could have something like:
```
install_requires=['numpy>=1.23.2', 'pandas>=1.5.0', 'xarray>=2022.6.0', 'toml', 'scipy>=1.9.1'],
```

Or should we state newer version to be on the safe side?

Reply:
Sure, that is probably fine. Could maybe even just do the clean minor versions rather than a patch version? Like `scipy>=1.9.0` rather than `scipy>=1.9.1`? And `numpy>=1.23.0`, rather than `numpy>=1.23.2`? But that would require downgrading and testing I suppose, so probably not worth it and doesn't matter.

Reply:
If I have time then I could do some testing.

Reply:
@PennyHow I went ahead and tested this. I removed and reinstalled packages so now I have:
```
numpy==1.23.0
pandas==1.5.0
scikit-learn==1.1.0
scipy==1.9.0
xarray==2022.6.0
```
Checked out to current `main` and running `./l3_processor.sh`, I get no errors or warnings. So, this should be good! I will push a change to `setup.py` to this branch.

Should we also include `netCDF4` in `install_requires=`?